datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -190,6 +202,32 @@ public SdxCluster getByCrn(String userCrn, String clusterCrn) {
         }
     }
 
+    public String getRuntimeVersionFromImageRequest(ImageSettingsV4Request imageSettingsV4Request) {
+        try {
+            LOGGER.info("Calling cloudbreak to get runtime version for the given image catalog {} and image id {}",
+                    imageSettingsV4Request.getCatalog(), imageSettingsV4Request.getId());
+            ImageV4Response imageV4Response =  ThreadBasedUserCrnProvider.doAsInternalActor(
+                    () -> {
+                        try {
+                            return imageCatalogV4Endpoint.getImageByNameAndId(0L, imageSettingsV4Request.getCatalog(), imageSettingsV4Request.getId());
```

Comment:
Not a big deal but could you please extract `0L` as a new constant `WORKSPACE_ID`?

Reply:
done